qcodes/instrument_drivers/AimTTi/AimTTi_PL601P_channels.py

```diff
@@ -0,0 +1,311 @@
+                           'Here, the integer 1 is for the Low range, '
+                           'and integer 2 is for the High range.')
+
+        self.add_parameter('curr_step_size',
+                           get_cmd=self._get_current_step_size,
+                           get_parser=float,
+                           set_cmd=f'DELTAI{channel} {{}}',
+                           label='Current Step Size',
+                           unit='A')
+
+        self.add_parameter('output',
+                           get_cmd=f'OP{channel}?',
+                           get_parser=float,
+                           set_cmd=f'OP{channel} {{}}',
+                           val_mapping=create_on_off_val_mapping(on_val=1,
+                                                                 off_val=0))
+
+    def _get_voltage_value(self) -> float:
+        channel_id = self.channel
+        _voltage = self.ask_raw(f'V{channel_id}?')
+        _voltage_split = _voltage.split()
+        return float(_voltage_split[1])
+
+    def _get_current_value(self) -> float:
+        channel_id = self.channel
+        _current = self.ask_raw(f'I{channel_id}?')
+        _current_split = _current.split()
+        return float(_current_split[1])
+
+    def _get_voltage_step_size(self) -> float:
+        channel_id = self.channel
+        _voltage_step_size = self.ask_raw(f'DELTAV{channel_id}?')
+        _v_step_size_split = _voltage_step_size.split()
+        return float(_v_step_size_split[1])
+
+    def _get_current_step_size(self) -> float:
+        channel_id = self.channel
+        _current_step_size = self.ask_raw(f'DELTAI{channel_id}?')
+        _c_step_size_split = _current_step_size.split()
+        return float(_c_step_size_split[1])
+
+    def _set_current_range(self, val: int) -> None:
+        """
+        This is the private function that ensures that the output is switched
+        off before changing the current range, as pointed out by the instrument
+        manual.
+        """
+        channel_id = self.channel
+        self.output(False)
+        self.write(f'IRANGE{channel_id} {val}')
+
+    def increment_volt_by_step_size(self) -> None:
+        """
+        A bound method that increases the voltage output of the corresponding
+        channel by an amount the step size set by the user via ``volt_step_size``
+        parameter.
+        """
+        channel_id = self.channel
+        self.write(f'INCV{channel_id}')
+        # Clear the cache.
+        _ = self._get_voltage_value()
+
+    def decrement_volt_by_step_size(self) -> None:
+        """
+        A bound method that decreases the voltage output of the corresponding
+        channel by an amount the step size set by the user via ``volt_step_size``
+        parameter.
+        """
+        channel_id = self.channel
+        self.write(f'DECV{channel_id}')
+         # Clear the cache.
+        _ = self._get_voltage_value()
```

Comment:
Does this update the `volt` parameter? If I do 
```python
aim.volt(4)
aim.decrement_by_step_size()
print(aim.volt.get_latest())
```
what do I then get? I think the `decrement_volt_by_step_size` should instead call `self.volt.get()`.

Reply:
This is a pretty good catch! 💯  Fixed. :)